src/features/auth/signup/SetEmail/SetEmail.native.test.tsx

```diff
@@ -9,6 +9,12 @@ describe('<SetEmail />', () => {
   beforeEach(jest.clearAllMocks)
   afterAll(jest.clearAllMocks)
 
+  it('should match previous snapshot', () => {
```

Comment:
je pense qu'il serait beaucoup intéressant de vérifier que lorsqu'on render la modal, on vérifie qu'il y ai un bouton avec un nom correct qui lorsqu'on clic dessus nous redirige vers la page de login

Reply:
J'ai fait ce test seul pour deux raisons :
- J'ai changé un élement du design de log in button (un espace qu'il manquait) et les tests passaient toujours même si le composant était utilisé dans une autre page (`LoggedOutHeade`r), j'ai trouvé que c'était pas top comme en plus il était pas dans storybook, si un jour la couleur passe plus à cause d'une refacto par exemple on sera pas vraiment protégés...
- Le comportement au clic est justement testé dans la page `LoggedOutHeader`, comme c'est exactement le même bouton je me suis dit que ça faisait redondant de le tester à nouveau

Je sais pas ce que tu en penses, je peux toujours copier le test du `LoggedOutHeader` si besoin, et tester le design autrement selon tes reco 🤓 je pense que ce serait top si on se mettait d'accord sur une stratégie de tests claire au niveau de la tribe !

Reply:
pour ce bouton, ça pourrait etre chouette de le tester unitairement